archivist/runner.py

```diff
@@ -358,24 +345,25 @@ def actions(self):
         return self._actions
 
     @property
-    def action(self) -> Callable:
+    def args(self):
```

Comment:
Added to enable unittesting of the if keywords is not None clause in the init_args method.